wishlist.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Arcade Store</title>
+    <script src="https://kit.fontawesome.com/4a9487b343.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://nifty-tesla-85dda2.netlify.app/Components/components.css">
+    <link rel="stylesheet" href="common.css">
+    <link rel="stylesheet" href="wishlist.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="nav-logo">
+            <img src="./Assets/nav-logo.png" alt="logo" />
+        </div>
+        <div class="nav-items">
+            <a class="nav-link sm-title hide-md" href="products.html">Shop</a>
+            <a class="nav-link sm-title hide-md" href="">Categories</a>
+        </div>
+        <div class="nav-space"></div>
+        <div class="nav-icon-items">
+            <div class="nav-search-bar hide-sm">
+                <input class="search-bar" type="text" placeholder="Search..">
+                <i class="fa fa-brands fa-searchengin"></i>
+            </div>
+            <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                <i class="fa fa-solid fa-user"></i>
+            </button>
+            <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                <i class="fa fa-solid fa-heart"></i>
+            </button>
+            <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                <i class="fas fa-shopping-cart"></i>
+            </button>
+            <button id="dark-bg-icon" class="btn icon-btn show-md">
```

Comment:
Maintain consistent naming of classes with component first and character next.
Example :
instead of icon-btn name it as btn-icon

Reply:
agreed but that will require a lot of refactoring in my component library. will do it in a later commit!